__unported__/sale_payment_method/sale.py

```diff
@@ -300,5 +296,5 @@ def action_cancel(self, cr, uid, ids, context=None):
                     _('Cannot cancel this sales order!'),
```

Comment:
It's very easy to convert this to new API with @api.multi and exceptions.Warning